modules/migration-creating-migration-plan-cam.adoc

```diff
@@ -51,24 +33,28 @@ You can use _direct image migration_ and _direct volume migration_ to migrate im
 . On the *Namespaces* page, select the projects to be migrated and click *Next*.
 . On the *Persistent volumes* page, click a *Migration type* for each PV:
 
-* The *Copy* option copies the data from the PV of a source cluster to the replication repository and then restores it on a newly created PV, with similar characteristics, in the target cluster.
-* The *Move* option unmounts a remote volume, for example, NFS, from the source cluster, creates a PV resource on the target cluster pointing to the remote volume, and then mounts the remote volume on the target cluster. Applications running on the target cluster use the same remote volume that the source cluster was using. The remote volume must be accessible to the source and target clusters.
+* The *Copy* option copies the data from the PV of a source cluster to the replication repository and then restores the data on a newly created PV, with similar characteristics, in the target cluster.
+* The *Move* option unmounts a remote volume, for example, NFS, from the source cluster, creates a PV resource on the target cluster pointing to the remote volume, and then mounts the remote volume on the target cluster. Applications running on the target cluster use the same remote volume that the source cluster was using.
 
 . Click *Next*.
 . On the *Copy options* page, select a *Copy method* for each PV:
 
 * *Snapshot copy* backs up and restores data using the cloud provider's snapshot functionality. It is significantly faster than *Filesystem copy*.
 * *Filesystem copy* backs up the files on the source cluster and restores them on the target cluster.
++
+The file system copy method is required for direct volume migration.
```

Comment:
_direct volume migration_
Does the average person doing the migration know what this is?

Reply:
Yes. It is described in an earlier section and in the prerequisites.